src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {

```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line (line 2) before the class declaration.